core/notification/router_receiver_service.go

```diff
@@ -51,10 +84,55 @@ func (s *RouterReceiverService) PrepareMetaMessages(ctx context.Context, n Notif
 		})
 	}
 
-	var metaMessagesNum = len(metaMessages)
-	if metaMessagesNum > s.deps.Cfg.MaxMessagesReceiverFlow {
-		return nil, nil, errors.ErrInvalid.WithMsgf("sending %d messages exceed max messages receiver flow threshold %d. this will spam and broadcast to %d channel. found %d receiver selectors passed, you might want to check your receiver selectors configuration", metaMessagesNum, s.deps.Cfg.MaxMessagesReceiverFlow, metaMessagesNum, len(n.ReceiverSelectors))
+	return metaMessages, notificationLogs, nil
+}
+
+func (s *RouterReceiverService) handleConfigCase(ctx context.Context, selectors []map[string]any) ([]receiver.Receiver, map[uint64]map[string]interface{}, error) {
+	var receiverIDs []uint64
+	userConfigs := make(map[uint64]map[string]interface{})
+
+	for _, selector := range selectors {
+		if idStr, ok := selector["id"].(string); ok {
+			id, err := strconv.ParseUint(idStr, 10, 64)
+			if err != nil {
+				return nil, nil, errors.ErrInvalid.WithMsgf("invalid receiver id: %s", idStr)
+			}
+			receiverIDs = append(receiverIDs, id)
```

Comment:
why do we need this?

Reply:
This part is converting "id" in uint64, while calling API we are passing ids as string.

Reply:
We dont need to do this right? The idea is all receiver evaluation is through label matching and id is one part of the label there.
Cmiiw